### ISSUE TITLE
typescript: typing errors are shallowed

### ISSUE THREAD

**Issue by Guria** [christianalfoni/webpack-bin#80]
There is no linting support and/or typescript compilation errors.
Seems it's just ignored.

**Comment by christianalfoni** [comment]
Yeah, there is not browser linter for TypeScript. There is one in ACE editor I believe though, so considering moving to that. Seems like ACE editor also lints in a worker environment which improves performance.

**Comment by christianalfoni** [comment]
Related to #72

**Comment by niieani** [comment]
Monaco solves this even better than ACE, cause not only does it lint, it actually does type-checking in the browser. See: https://github.com/christianalfoni/webpack-bin/issues/158
